queue.go

```diff
@@ -509,3 +509,29 @@ func (q *Queue) updateQueuePath() error {
 
 	return nil
 }
+
+func (q *Queue) updateCurrentSong(oldAudio *AudioFile, newAudio *AudioFile) (err error) {
```

Comment:
What this function does? I dont understand the intention also returning error for `updateCurrentQueue` is redundant here because there is no possible error being returned inside the body

Reply:
> What this function does? I dont understand the intention also returning error for `updateCurrentQueue` is redundant here because there is no possible error being returned inside the body

This function update the current playing song to reflect changes cause by rename and paste.  The error return is not necessary so I'll remove it.